File: guess.cpp
#include<iostream>
using namespace std;
int main(){
int password = 1237;

int guess = 0;
while(guess != password){
cout << "Guess the password : ";
cin>>guess;
if(guess == password){
cout << "Correct Guess\n";
break;
}
else if(guess > password)
    cout << "Lower\n";
else
    cout << "Higher\n";
}
//MyReadFile.close();
system("pause");
return 0;
}

File: guess.py
import random as r

def guess_password(password):
    try:
        n = int(input('Guess the password : '))
        while n != password:
            if n > password:
                print('Lower.')
            else:
                print('higher')
            n = int(input('Guess the password : '))
        print('Correct Guess.')
    except Exception as e:
        print('Invalid integer')

guess_password(543212)
